### PR TITLE
Fix error alerts, drop rpc getResource method

### PR DESCRIPTION
**Pull request by nikitayutanov**
Bumps deps and fixes error alerts messages display

### CODE REVIEW

File: website/frontend/src/services/ServerRPCRequestService.ts [hunk 1]
```diff
@@ -24,18 +24,12 @@ type RPCRequest = {
...
-  data: string;
-};
-
 // eslint-disable-next-line @typescript-eslint/naming-convention
 type _RPCResponse<Result> = {
   id: JsonRpcId;
   jsonrpc: JsonRpcVersion;
   result: Result;
-  error: RPCError;
+  error: string;
```
Comment (codev0) [inline website/frontend/src/services/ServerRPCRequestService.ts:32]: Its not by specification https://www.jsonrpc.org/specification#response_object

Comment (nikitayutanov) [reply website/frontend/src/services/ServerRPCRequestService.ts:32]: Thanks, gonna be resolved on backend.

Comment (nikitayutanov) [reply website/frontend/src/services/ServerRPCRequestService.ts]: Fixed.

File: website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx [hunk 2]
```diff
@@ -27,12 +27,12 @@ export const ProgramSwitch: VFC<Props> = ({ pageType }) => {
         throw new Error(`WALLET NOT CONNECTED`);
       }
 
-      const response = await apiRequest.getResource(RPC_METHODS.GET_TEST_BALANCE, {
+      const response = await apiRequest.callRPC(RPC_METHODS.GET_TEST_BALANCE, {
         address: `${currentAccount.address}`,
       });
 
       if (response.error) {
-        alert.error(`${response.error.error}`);
```
Comment (osipov-mit) [inline website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx:35]: Is there a way to reach to this line? If there will be an error in the response, `callRpc` method will throw an error, I suppose

File: website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx [hunk 3]
```diff
@@ -27,12 +27,12 @@ export const ProgramSwitch: VFC<Props> = ({ pageType }) => {
...
+        alert.error(`${response.error.message}`);
```
Comment (osipov-mit) [inline website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx:35]: Is there a way to reach to this line? If there will be an error in the response, callRpc method will throw an error, I suppose

Comment (nikitayutanov) [reply website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx:35]: Alert gets called on error afterall, but I agree that it's a weird way to catch it and should be refactored.

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**